### ISSUE TITLE
Object.entries is not a function on node <= 6

### ISSUE THREAD

**Issue by TheDancingCode** [cloudinary/cloudinary_npm#219]
Version `1.12.0` uses `Object.entries` in `lib-es5` although it isn't an es5 feature. This generates the following error, using node 6 in my case:
```
/node_modules/cloudinary/lib-es5/config.js:64
      Object.entries(parsedConfig).forEach(function (_ref) {
             ^
TypeError: Object.entries is not a function
```
An easy fix is to use the [transform-es2017-object-entries](https://github.com/christensson/babel-plugin-transform-es2017-object-entries) plugin in the babel config. The reason your tests do pass is because you're using `babel-polyfill` in the test command. For the tests to be representative, they should work with the same babel config as is used to compile `lib-es5`.

**Comment by cloudinaraz** [comment]
Thank you for bringing this to our attention @TheDancingCode, we will fix this shortly.

**Comment by joechoi-git** [comment]
This issue caused our Cloudinary integration to break. We will rollback to cloudinary@1.11.0. Please be careful with these updates.

**Comment by tocker** [comment]
Fixed in version 1.13.1
